pyproject.toml

```diff
@@ -76,7 +76,7 @@ lint.ignore = ["D104", "PLR2004", "RUF200"]
 
 [tool.ruff.lint.per-file-ignores]
 # S603 unchecked input in subprocess call is fine in our tests
-"*/tests/*" = ["D", "S101", "S603"]
+"*/tests/*" = ["D", "S101", "S603", "S311"]
```

Comment:
We could probably add this to the list of `lint.ignore`, not just for tests. I can't foresee a case where we would be concerned about this, but maybe that is naive 😅

Reply:
I totally missed that. Yes, that makes senses to go there.